src/main/java/edu/kis/powp/jobs2d/command/ComplexDriverCommand.java

```diff
@@ -0,0 +1,30 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.util.Iterator;
+import java.util.List;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.visitor.ICommandVisitor;
+
+public class ComplexDriverCommand implements ICompoundCommand {
+    List<DriverCommand> commands;
```

Comment:
Should be private. 
BTW: You could avoid implementing this class by adding a default accept implementation in ICompoundCommand